dashboards/google-anthos/anthos-cluster-control-plane-uptime.json

```diff
@@ -176,7 +176,14 @@
               "plotType": "LINE",
               "targetAxis": "Y1",
               "timeSeriesQuery": {
-                "timeSeriesQueryLanguage": "{ fetch k8s_container\n| metric\n    'kubernetes.io/anthos/apiserver_admission_webhook_admission_duration_seconds'\n| add [cluster: resource.cluster_name, namespace: resource.namespace_name]\n| align delta(1m)\n| every 1m\n| group_by\n    [resource.project_id, resource.location, cluster, namespace],\n    [value_percentile50:\n       percentile(value.apiserver_admission_webhook_admission_duration_seconds,\n         50)]\n; fetch prometheus_target\n| metric\n    'kubernetes.io/anthos/apiserver_admission_webhook_admission_duration_seconds/histogram'\n| add [cluster: resource.cluster, namespace: resource.namespace]\n| align delta(1m)\n| every 1m\n| group_by\n    [resource.project_id, resource.location, cluster, namespace],\n    [value_percentile50:\n       percentile(value.histogram,\n         50)] }\n| union"
+                "prometheusQuery": "# 50th percentile from k8s_container source\nhistogram_quantile(\n  0.5,\n  sum by (project_id, location, cluster_name, namespace_name, le) (\n    rate(kubernetes_io:apiserver_admission_webhook_admission_duration_seconds_bucket{monitored_resource=\"k8s_container\"}[1m])\n  )\n)"
+              }
+            },
+            {
+              "plotType": "LINE",
+              "targetAxis": "Y1",
+              "timeSeriesQuery": {
+                "prometheusQuery": "# 50th percentile from prometheus_target source\nhistogram_quantile(\n  0.5,\n  sum by (project_id, location, cluster, namespace, le) (\n    rate(kubernetes_io:apiserver_admission_webhook_admission_duration_seconds_histogram_bucket{monitored_resource=\"prometheus_target\"}[1m])\n  )\n)"
```

Comment:
I get an error with `prometheus_target` specified explicitly. For all the `prometheus_target` ones, just remove the whole `{monitored_resource=\"prometheus_target\"}` then it works